libs/sql-schema-describer/src/postgres.rs

```diff
@@ -637,20 +649,26 @@ impl SqlSchemaDescriber {
                 Some(x) if x.starts_with("NULL") => None,
                 Some(default_string) => {
                     Some(match &tpe.family {
-                        ColumnTypeFamily::Int => match Self::parse_int(&default_string) {
-                            Some(int_value) => DefaultValue::value(int_value),
-                            None => match is_autoincrement(&default_string, sequences) {
-                                Some(seq) => DefaultValue::sequence(seq),
-                                None => DefaultValue::db_generated(default_string),
-                            },
-                        },
-                        ColumnTypeFamily::BigInt => match Self::parse_big_int(&default_string) {
-                            Some(int_value) => DefaultValue::value(int_value),
-                            None => match is_autoincrement(&default_string, sequences) {
-                                Some(seq) => DefaultValue::sequence(seq),
-                                None => DefaultValue::db_generated(default_string),
-                            },
-                        },
+                        ColumnTypeFamily::Int | ColumnTypeFamily::BigInt => {
+                            let default_expr = unsuffix_default_literal(
+                                &default_string,
+                                &[data_type, &tpe.full_data_type, "integer", "INT8", "INT4"],
+                            )
+                            .unwrap_or_else(|| default_string.as_str().into());
+                            let default_expr = process_string_literal(&default_expr);
+
+                            match default_expr.parse::<i64>().ok() {
```

Comment:
Can the default expression be more than the `i64::MAX`?

Reply:
On postgres/cockroach `i64` is the largest integer column type, and there aren't unsigned integers, so I think it can't be bigger.